Makefile

```diff
@@ -269,6 +269,37 @@ work/stunnel.conf:
 	@echo cafile=$(ROOT_DIR)/work/ca/certs/ca.cert.pem >> $@
 	@echo verify=2 >> $@
 
+	@echo [stunnel-master-slave-node-1] >> $@
+	@echo accept = 127.0.0.1:8443 >> $@
+	@echo connect = 127.0.0.1:6482 >> $@
+	@echo cert=$(ROOT_DIR)/work/ca/certs/localhost.cert.pem >> $@
+	@echo key=$(ROOT_DIR)/work/ca/private/localhost.decrypted.key.pem >> $@
+	@echo capath=$(ROOT_DIR)/work/ca/certs/localhost.cert.pem >> $@
+	@echo cafile=$(ROOT_DIR)/work/ca/certs/localhost.cert.pem >> $@
+
+	@echo [stunnel-master-slave-node-2] >> $@
+	@echo accept = 127.0.0.1:8444 >> $@
+	@echo connect = 127.0.0.1:6483 >> $@
+	@echo cert=$(ROOT_DIR)/work/ca/certs/localhost.cert.pem >> $@
+	@echo key=$(ROOT_DIR)/work/ca/private/localhost.decrypted.key.pem >> $@
+	@echo capath=$(ROOT_DIR)/work/ca/certs/localhost.cert.pem >> $@
+	@echo cafile=$(ROOT_DIR)/work/ca/certs/localhost.cert.pem >> $@
+
+	@echo [stunnel-master-slave-client-cert-node-1] >> $@
```

Comment:
Let's skip the cert-auth nodes. Certificate-based authentication isn't something we have to test again.